<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProgettoGestione</class>
 <widget class="QMainWindow" name="ProgettoGestione">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1251</width>
    <height>892</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ProgettoGestione</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="welcomeLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>131</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AvantGarde LT Medium</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Benvenuto! </string>
    </property>
   </widget>
   <widget class="QTextEdit" name="userQuery">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>471</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit{
border: 2px solid;
border-radius:10px;
background-color: palette(base);
}</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>310</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AvantGarde LT Medium</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Inserisci la query qui sotto:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>130</y>
      <width>81</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border: 2px solid;
border-radius: 10px;
border-style: outset;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../../.designer/backup/images/search.jpg</normaloff>../../../../../.designer/backup/images/search.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QComboBox" name="genreList">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>42</y>
      <width>191</width>
      <height>30</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::PreventContextMenu</enum>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox{
border: 2px solid;
border-radius: 10px;
}


QComboBox:on {
    border-bottom-left-radius: 0px;
    border-bottom-right-radius: 0px;
}

QComboBox QAbstractItemView {
    border-bottom-right-radius: 10px;
    border-bottom-left-radius: 10px;
    background: white;
    border: 1px solid gray;
    box-shadow: transparent;
    padding: 4px 4px 4px 4px
}</string>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>40</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AvantGarde LT Medium</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Scegli un genere:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>681</width>
      <height>541</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextBrowser{
border: 2px solid;
background-color: palette(base);
border-radius: 20px;
}</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="querySyntax">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>210</y>
      <width>391</width>
      <height>281</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit{
border: 2px solid;
background-color: palette(base);
border-radius: 20px;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>170</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AvantGarde LT Medium</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Inserisci query:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1251</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
